Adds the Scav-class Drifter

Sylphily opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
adds the scav class ship, designed for salvaging ruins, but made from long ago, and has worn bones.
it is a ISV faction
![scavmk1](https://user-images.githubusercontent.com/30062054/167377762-eef229ab-06cc-4a9f-a64f-5ceb06b9a32d.png)
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
cool ship
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Scav-classed Drifter 
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sylphily commented:
fuck forgot desc

Sylphily commented:
any one knows why this is failing?

Fikou reviewed (changes requested):
this needs to use the TGM map format

Sylphily committed:
scav

angleless

hotfix

Update scav.dmm

Update scav.dmm

Update scav.dmm

Sylphily committed:
Update scav.dmm

Sylphily committed:
Update scav.dmm

Sylphily committed:
scav

angleless

hotfix

Update scav.dmm

Update scav.dmm

Update scav.dmm

Update scav.dmm

Update scav.dmm

Sylphily committed:
Merge remote-tracking branch 'origin/Salvagee' into Salvagee

Sylphily commented:
Yes I know the bridge has no wide band

ZephyrTFA commented:
Try to avoid having thin diagonal walls or windows

ZephyrTFA commented:
![image](https://user-images.githubusercontent.com/12817816/167462329-d321842e-5bc1-4bc5-be3f-8021c9b0a868.png)

Sylphily commented:
Literally can't without losing it's shape and keep room

PiperDoots commented:
> Literally can't without losing it's shape and keep room

try adding one one in the middle and then another next to it
![image](https://user-images.githubusercontent.com/90694223/167462748-1b5fdab6-8899-4ffb-b1a9-9018597fcf18.png)

Sylphily commented:
Then it double on thickness and looks more wonky

Sylphily commented:
Guess I can axe that extra seat

Sylphily committed:
Update scav.dmm

Sylphily committed:
Update scav.dmm

Sylphily committed:
Update scav.dmm

Sylphily commented:
i have no clue why its failing it seems to work and all the pipes are lined up

Apogee-dev commented:
Config file needs prefix and namelists fields. You also don't have any firelocks on the bridge (though honestly, I'd suggest placing firelocks on all of the dividing windows, considering how easy they are to break)

Apogee-dev commented on _maps/configs/scav.json:
```diff
@@ -0,0 +1,27 @@
+{
+	"map_name": "Scav-class Drifter",
+	"map_short_name": "Scav-class",
```
"prefix": ISV,
"namelists": insert your namelists of choice here, i'd suggest GENERAL and SPACE at a minimum

Sylphily commented:
and i had just closed DMM pain

Apogee-dev commented:
You also need to replace all your space tiles with terrain passthrough tiles (and the appropriate area). As a side note, unless they're placed over a floor tile with a ship area, the ship's exterior cameras will simply get deleted.

Sylphily commented:
camreas have been removed because they didn't seem to work any ways

Sylphily commented:
also where the hell is the file containing ship names, it dosen't seem to exist?

Apogee-dev commented:
> also where the hell is the file containing ship names, it dosen't seem to exist?

strings/ship_names.json

Sylphily commented:
ah i was on the wrong branch, couldn't find them there lol

Sylphily committed:
names, firelocks

Sylphily committed:
atmos=good

TerraTheUnwise commented:
I very much enjoy this ship. I played a round with it. And looting ruins was very quick and efficient.

Sylphily committed:
Update scav.dmm

Latentish commented:
I feel like this ship and the Masinyane serve very similar functions. What makes this one special?

Sylphily commented:
> I feel like this ship and the Masinyane serve very similar functions. What makes this one special?

well it is more scavenger focused, and is just more thematic to scavenging, it has room to RP and room for a bit of medical if you know what you are doing, it also has cargo space for what ever you want, and is quite open, and comfy
